tests/plot_test.py

```diff
@@ -0,0 +1,144 @@
+"""Test plot
+
+:copyright: Copyright (c) 2025 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+from pykern import pkunit
```

Comment:
generally we don't import it here just out of formality so that all tests defer imports, which can be very important.